CHANGELOG.rst

```diff
@@ -4,6 +4,19 @@ Community Windows Release Notes
 
 .. contents:: Topics
 
+v3.0.2
+======
+
+Release Summary
+---------------
+
+Release summary for v3.0.2
+
+Bugfixes
+--------
+
+- win_initialize_disk - disks that are formatted but offline cannot be brought online without erasing them (https://github.com/ansible-collections/community.windows/issues/149).
```

Comment:
We don't touch the `CHANGELOG.rst` directory but create a "fragment" file under https://github.com/ansible-collections/community.windows/tree/main/changelogs/fragments. The docs under https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to hopefully give you an idea of what that looks like.